src/libraries/System.Private.CoreLib/src/System/Text/Utf8Span.Conversion.cs

```diff
@@ -43,7 +43,11 @@ public int Normalize(Span<byte> destination, NormalizationForm normalizationForm
         {
             // TODO_UTF8STRING: Reduce allocations in this code path.
 
-            ReadOnlySpan<char> normalized = this.ToString().Normalize(normalizationForm);
+            ReadOnlySpan<char> normalized = this.ToString().Normalize(normalizationForm)
+#if NETSTANDARD2_0
```

Comment:
The ifdef isn't needed. May as well call `AsSpan` on all platforms; it's not going to harm anything.

(Same comment applies to ifdefed `AsSpan` calls elsewhere in this method.)

Reply:
Ok, I wasn't sure if it was going to cause issues. Will remove the ifdefs.